tests/unit/lms/validation/authentication/_bearer_token_test.py

```diff
@@ -103,23 +107,14 @@ def schema(self, pyramid_request):
         return BearerTokenSchema(pyramid_request)
 
     @pytest.fixture
-    def pyramid_request(self, _jwt):
-        pyramid_request = DummyRequest()
-        pyramid_request.headers[
-            "authorization"
-        ] = f"Bearer {_jwt.encode_jwt.return_value}"
+    def pyramid_request(self, pyramid_request):
+        pyramid_request.headers["authorization"] = "Bearer ENCODED_JWT"
         return pyramid_request
 
 
 @pytest.fixture(autouse=True)
-def _jwt(patch, lti_user):
-    _jwt = patch("lms.validation.authentication._bearer_token._jwt")
-    _jwt.encode_jwt.return_value = "ENCODED_JWT"
-    _jwt.decode_jwt.return_value = lti_user._asdict()
-    return _jwt
-
+def jwt_service(jwt_service, lti_user):
+    jwt_service.decode_with_secret.return_value = lti_user._asdict()
+    jwt_service.encode_with_secret.return_value = "ENCODED_JWT"
```

Comment:
I know it's a point of contention, but when a fixture is only used in one place I prefer it to be scoped as close to that place as possible. This is partly a general preference for local vs. global, but also for the nuts and bolts reason that it makes refactoring in future much easier.

If you have 4 test classes and a soup of fixtures at the bottom, it's more difficult to work out what's shared and what's not to break them up. If the fixtures are part of the classes they are used in, then it's easy to snip out the whole test class.